packages/SymbioticController/composer.json

```diff
@@ -4,7 +4,8 @@
         "php": "^7.1",
         "nette/di": "^2.4",
         "nette/application": "^2.4",
-        "odbav-to/presenter-route": "^0.1"
+        "odbav-to/presenter-route": "^0.1",
```

Comment:
I agree with you. I prefer `PresenterRoute` because it's simple and it requires 2nd argument to be invokable presenter with validation.

To be honest, I planned to put this route here, since it is single class that is required to make it work. But you were faster :D 

Would you be ok with adding that class in here?